### DISCUSSION TITLE
opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ]

### DISCUSSION THREAD

**Discussion by devvsakib** [webpack/webpack#16355] [Q&A]
# Bug report

**What is the current behavior?**
```
node:internal/crypto/hash:71
  this[kHandle] = new _Hash(algorithm, xofLen);
  opensslErrorStack: [ 'error:03000086:digital envelope routines::initialization error' ],
  library: 'digital envelope routines',
  reason: 'unsupported',
  code: 'ERR_OSSL_EVP_UNSUPPORTED'
}
```
first server run, it occurs when I reload the page. 
then I use `export NODE_OPTIONS=--openssl-legacy-provider`. this solves the error but problem is, when I reopen my code editor after closing, it happens again. I have tried all the possible ways to fix it but failed.

**Other relevant information:**
webpack version:  5
Node.js version: 18.8.0
Operating System: Windows 11 x64

**Comment by alexander-akait** [comment]
It means you still use `md4` somewhere, double check webpack-cli and webpack version

**Comment by olof-nord** [comment]
The problem is that many plugins are using md4 by default. So far, I have found 'file-loader', and 'compression-webpack-plugin' which defaults to this algorithm by default - especially if using webpack 4.
